wdl2cwl/tests/wdl_files/align_and_count.wdl

```diff
@@ -0,0 +1,21 @@
+version 1.0
+
+# Source: https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/workflows/align_and_count.wdl
+
+import "/home/dennis/Documents/CWLab/wdl-cwl-translator/wdl2cwl/tests/wdl_files/tasks/tasks_reports.wdl" as reports
+# import "../tasks/tasks_reports.wdl" as reports
```

Comment:
```suggestion
import "./tasks/tasks_reports.wdl" as reports
```
In unix / shell `..` means the parent directory, but the `tasks` folder is in THIS folder (`.`)